crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -251,7 +250,7 @@ $env.config = {
         env_change: {
             PWD: [{|before, after| null }] # run if the PWD environment is different since the last repl input
         }
-        display_output: { table } # run before the output of a command is drawn, example: `{ if (term size).columns >= 100 { table -e } else { table } }`
+        display_output: "if (term size).columns >= 100 { table -e } else { table }" # run to display the output of a pipeline
```

Comment:
just for my understanding :yum: 
do we *need* to have this a string instead of a closure as before?

Reply:
We could put a closure here, but that'd be different than the default, which uses a string. We use a string on the nushell side because creating a closure when you're not in parse time is difficult, but a string is much easier.

Reply:
make sense for the default value, just wanted to make sure the closure way was still possible :relieved: